src/screens/rewards/components/RewardsProgramStatus.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Box, Stack, Text } from '@/design-system';
+import { useDimensions } from '@/hooks';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+type Props = {
+  status: 'error' | 'finished';
+};
+
+export const RewardsProgramStatus: React.FC<Props> = ({ status }) => {
+  const { height } = useDimensions();
+  const { top } = useSafeAreaInsets();
+
+  const emoji = status === 'error' ? '😵' : '💸';
+  const title = status === 'error' ? 'Something went wrong' : 'Program ended';
+  const text =
+    status === 'error'
+      ? 'Please check your internet connection and check back later.'
+      : 'Stay tuned for what we have in store for you next!';
+
+  return (
+    <Box
+      width="full"
+      height={{ custom: height - top - 40 }}
```

Comment:
what is this magic number

Reply:
I will make it more explicit, bascially it's the 20px padding around the card times two

Reply:
It was double outer sheet content padding (20px) But it wasn't needed